porter/v1/cluster_control_plane.proto

```diff
@@ -1353,6 +1392,12 @@ service ClusterControlPlaneService {
   // and runs the provided command, or if no command is provided, runs the command defined for the service.
   rpc ManualServiceRun(ManualServiceRunRequest) returns (ManualServiceRunResponse) {}
 
+  // JobRunStatus returns the status of a given job run
+  rpc JobRunStatus(JobRunStatusRequest) returns (JobRunStatusResponse) {}
+
+  // JobRuns returns the job runs for a given app and job service in the provided deployment target
+  rpc JobRuns(JobRunsRequest) returns (JobRunsResponse) {}
```

Comment:
thoughts on making this one endpoint? And if we pass in the `job_run_name` then we only return one?

Reply:
I'm leaning towards keeping them separate for now, so that anyone who wants to wrap our API can expect a consistent return type for each operation (get, list)